xbmc/addons/AddonManager.h

```diff
@@ -380,10 +393,13 @@ namespace ADDON
 
     VECADDONS m_updateableAddons;
 
+    VECADDONS GetAvailableUpdatesOrOutdatedAddons(bool returnOutdatedAddons) const;
```

Comment:
A bit doc missing about.
And maybe to use direct the `std::vector<AddonPtr>`, I think to kickout the `VECADDONS` as with vector more clean to see what brings, or maybe as `std::vector<std::shared_ptr<IAddon>>` (as you done on other places).